plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/AbstractSQLDialect.java

```diff
@@ -741,6 +741,9 @@ public String getColumnTypeModifiers(@NotNull DBPDataSource dataSource, @NotNull
                         return null;
                     }
                     return "(" + maxLength + ")";
+                } if ("VARCHAR".equals(typeName)) {
```

Comment:
Please remove it, it breaks entire logic.
+ No constant
+ No `else`

Reply:
yeah, forgot "else"